js/preview/src/index.ts

```diff
@@ -1,5 +1,16 @@
-import { ChildProcess, spawn } from "node:child_process";
-import { create_server } from "./dev_server";
```

Comment:
Should we delete dev_server.ts?

Reply:
Oh, i thought we did. Must have some back when I merged main.